tests/resources/mailmap/.gitted/hooks/applypatch-msg.sample

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
```

Comment:
You should probably remove all these hooks just to keep the churn as small as necessary